lib/weaviate/schema.rb

```diff
@@ -27,6 +27,8 @@ def create(
       description: nil,
       properties: nil,
       multi_tenant: nil,
+      auto_tenant_creation: nil,
+      auto_tenant_activation: nil,
```

Comment:
I feel like it would be simpler to just default to `false` as that's what Weaviate itself does:
```ruby
    multi_tenant: false,
    auto_tenant_creation: false,
    auto_tenant_activation: false
```

And we'd just always set those values:
```ruby
req.body["multiTenancyConfig"] { enabled: multi_tenant, autoTenantCreation: auto_tenant_creation, autoTenantActivation: auto_tenant_activation
```

Reply:
I was thinking that it was strange that multi_tenant was the only arg that didn't match the weaviate arg, but didn't want to make a whole lot of changes in this PR. 

```ruby
MultiTenancyConfig = Struct.new(:enabled, :autoTenantCreation, :autoTenantActivation) do
  def initialize(enabled: false, autoTenantCreation: false, autoTenantActivation: false)
    super
  end
end
```

```ruby
create(
  ... 
  multi_tenancy_config: MultiTenancyConfig.new
)
```

Since this can be used in `update` as well, I think it'd make sense to have a small Struct that defines the options.

For now, I'll go with your suggestion, because we are way off track of fixing the present? / blank? issue

Reply:
So, just to be sure, you want me to blow away the existing code for this?

Reply:
Yes, please! 

I think it's okay, I'll release a 0.10 to signify the breaking change here.